bumble/smp.py

```diff
@@ -522,7 +522,7 @@ async def confirm(self) -> bool:
     async def compare_numbers(self, number: int, digits: int) -> bool:
         return True
 
-    async def get_number(self) -> int:
+    async def get_number(self) -> Optional[int]:
```

Comment:
For sure. Could you add a comment to specify that returning None means that the delegate was unable to return a valid number?

Reply:
Done! But I'm not sure it's correctly expressed.

Reply:
lgtm